content/05.conclusion.md

```diff
@@ -1,18 +1,26 @@
 ## Discussion and Conclusions
 
 BioRxiv is a constantly growing repository that contains life science preprints.
-The majority of research involving bioRxiv focuses on the metadata of preprints; however, the language contained within these preprints has not previously been systematically examined.
+Over 77% of bioRxiv preprints that were published ended up within Pubmed Central's Open Access Corpus (PMCOA) which is a significant improvement compared to the earlier years [@doi:10.7287/peerj.preprints.3520v1].
```

Comment:
```suggestion
Over 77% of bioRxiv preprints that were published ended up within Pubmed Central's Open Access Corpus (PMCOA) [@doi:10.7287/peerj.preprints.3520v1].
```
Is this necessarily true? I thought we knew they were published, but some might be linked to sources _not_ in pubmed but that are captured by bioRxiv and represented in the crossref database?